db/controllers/controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("../models/models");
+
+exports.getTopics = (req,res)=>{
+    fetchTopics().then((topics)=>{
+        res.status(200).send({topics:topics});
+    })
+}
```

Comment:
Where we have a matching key value pair you can send this back as simply {topics} and node will interpret this as {topics: topics}.